src/server.rs

```diff
@@ -755,7 +803,7 @@ fn should_send_mapping(
         return false;
     }
 
-    signature.is_added() || ticks.is_new_for_client(entity)
+    signature.is_added() || !ticks.entities.contains_key(&entity)
```

Comment:
Don't like removing clarity like this. Keep the method or use `ClientTicksEntity.is_new()`.

Reply:
Same as for `contains` above.